src/components/calculator/index.js

```diff
@@ -1,9 +1,8 @@
 import { getSetData } from "../../api";
 
```

Comment:
If this isn't a component, maybe move it to something like './src/utils/calculator/index.js'

Reply:
Done!